Tests/Unit/Model/SepaCreditTransferPaymentMethodTest.php

```diff
@@ -51,4 +62,17 @@ public function testIsMerchantOnly()
     {
         $this->assertTrue($this->_oPaymentMethod->isMerchantOnly());
     }
+
+    public function testAddPostProcessingTransactionData()
+    {
+        $oTransaction = new SepaCreditTransferTransaction();
+        $oParentTransaction = oxNew(Transaction::class);
+        $oParentTransaction->loadWithTransactionId('transaction 1');
+
+        try {
+            $this->_oPaymentMethod->addPostProcessingTransactionData($oTransaction, $oParentTransaction);
```

Comment:
You can also just use `assertNull()` on the return value of the function to see if there were no errors. That way you can skip the try/catch block.

Reply:
👍